src/simplify_rules.jl

```diff
@@ -66,7 +66,24 @@ let
         @acrule(cot(~x)^2 + -1*csc(~x)^2 => one(~x))
         @acrule(cot(~x)^2 +  1 => csc(~x)^2)
         @acrule(csc(~x)^2 + -1 => cot(~x)^2)
-    ]
+
+        # double-angle rules
+        @acrule(cos(~x)^2 - sin(~x)^2   => cos(2 * ~x))
+        @acrule(2*sin(~x)cos(~x)        => sin(2 * ~x))
```

Comment:
```suggestion
        @acrule(2*sin(~x)*cos(~x)        => sin(2 * ~x))
```
Explicit `*` seem to be needed here to get the simplification to happen.